Makefile

```diff
@@ -152,7 +152,7 @@ test-python-integration-local: ## Run Python integration tests (local dev mode)
 test-python-integration-rbac-remote: ## Run Python remote RBAC integration tests
 	FEAST_IS_LOCAL_TEST=True \
 	FEAST_LOCAL_ONLINE_CONTAINER=True \
-	python -m pytest --tb=short -v -n 8 --color=yes --integration --durations=10 --timeout=1200 --timeout_method=thread --dist loadgroup \
+	python -m pytest --tb=short -v -n 4 --color=yes --integration --durations=10 --timeout=1200 --timeout_method=thread --dist loadgroup \
```

Comment:
Why 4?

Reply:
Currently with 8, runner is intermittently throwing `Error: The operation was canceled.` (not related to this PR, happening on all PRs) due to high memory issues in node. For stability, reduced to 4.